packages/hardhat/hardhat.config.js

```diff
@@ -66,15 +66,15 @@ module.exports = {
 
       */
     },
-        ////////////////////////////////WARNING: RINKEBY IS  DEPRECATED /////////////////////////////////////////////////
-
-//     rinkeby: {
-//       url: "https://rinkeby.infura.io/v3/460f40a260564ac4a4f4b3fffb032dad", // <---- YOUR INFURA ID! (or it won't work)
-//       //    url: "https://speedy-nodes-nyc.moralis.io/XXXXXXXXXXXXXXXXXXXXXXX/eth/rinkeby", // <---- YOUR MORALIS ID! (not limited to infura)
-//       accounts: {
-//         mnemonic: mnemonic(),
-//       },
-//     },
+    /// /////////////////////////////WARNING: RINKEBY IS  DEPRECATED /////////////////////////////////////////////////
```

Comment:
I usually prefer removing (instead of commenting out), but in this case I think commenting is better.

We can remove it from the code when rinkeby/ropsten stop working for good.